tool/checkstyle/rules.bzl

```diff
@@ -131,11 +135,15 @@ checkstyle_test = rule(
         ),
         "target": attr.label(
             doc = "The java_library target to check sources on",
```

Comment:
same, no more "java" here or in the next attribute

Reply:
fixed